### PR TITLE
Updated learning rate and warmup_learning_rate

### PR DESCRIPTION
**Pull request by jvishnuvardhan**
Looks like this is a typo. All the other models uses small learning rate excepti this specific model. As the user mentioned below, the loss spikes with this learning rate.

Updated learning rate and warmup_learning_rate based on the following GH issue

Fixes https://github.com/tensorflow/models/issues/10509

### CODE REVIEW

**Comment by pkulzc** [comment]
Are you training using a TPU v2-8 or v3-8? This config is tuned for that setup and the large learning rate is desired.
